### DISCUSSION TITLE
Error installing Root Patches

### DISCUSSION THREAD

**Discussion by cjohn12345** [community/community#48318] [Actions]
### Select Topic Area

Product Feedback

### Body

I have a iMac27.   Identifier iMac13,2. I am using OCLP 0.6.1.  OpenCore Legacy Patcher keeps telling me that I need to apply root patches.  I have installed them several time and still gets the error.  
![OCLP Root Error](https://user-images.githubusercontent.com/81331788/220750298-314e61e9-e592-42ce-95bb-0dadcf6abfe0.png)
[OCLP Root Error PDF.pdf](https://github.com/community/community/files/10808042/OCLP.Root.Error.PDF.pdf)

How can I get this resolved?

**Comment by jsoref** [comment]
This is not the right forum for support for OpenCore-Legacy-Patcher.

You probably want to file an issue/open a discussion near: https://github.com/dortania/OpenCore-Legacy-Patcher/

**Reply by Puntweb** [reply]
I can confirm I also have the same problem. I have a slightly different error. But also on an iMac 13,2 (27 inch late 2012). Tryed many times and unable to apply root patch with 0.6.2.

![Captura de pantalla 2023-03-29 a les 10 36 48](https://user-images.githubusercontent.com/110167082/228476446-975f0b22-2648-4275-b5eb-9c5f0185d6bb.png)

I checked the discusions forums and open issues, but they are closed for posting.

**Comment by jsoref** [comment]
The issue tracker is closed for a reason:
https://github.com/dortania/OpenCore-Legacy-Patcher/issues/985

The repository does include instructions for getting community support:
https://github.com/dortania/OpenCore-Legacy-Patcher#support

You really should start by reading projects' readme files and searching their open issues.

None of that justifies hijacking some other random issue tracker/discussion forum (which in turn has its own readme which you also aren't reading).

**Reply by Puntweb** [reply]
My apologies. You're absolutely right!

I don't have much experience in Github, and I was a little confused on were to look for help or report problems. Now I've looked at the information more carefully, and noticed my post was out of place. Thanks a lot for your links and information.
